packages/vat-data/test/test-is-instance-virtual-class-kits.js

```diff
@@ -0,0 +1,106 @@
+// modeled on test-is-instance-heap-class-kits.js
```

Comment:
test-is-instance-heap-class-kits.js only arrived on endo master after the last sync with agoric-sdk.